STATUS: INLINE FAIL SETUPTEARDOWN
MODIFIED: de.strullerbaumann.visualee.dependency.boundary.DependencyContainerTest.testGetRelevantClassesDirectlyConnected
MODIFIED FILE: /home/alex/Repositories/visualee/src/test/java/de/strullerbaumann/visualee/dependency/boundary/DependencyContainerTest.java
CLEANER: de.strullerbaumann.visualee.examiner.ExaminerTest.testCleanPrimitives
CLEANER FILE: /home/alex/Repositories/visualee/src/test/java/de/strullerbaumann/visualee/examiner/ExaminerTest.java
POLLUTER: de.strullerbaumann.visualee.filter.entity.PackageFilterTest.testIsOkIncludeTrue
POLLUTER FILE: /home/alex/Repositories/visualee/src/test/java/de/strullerbaumann/visualee/filter/entity/PackageFilterTest.java
ORIGINAL CLEANER SIZE: 45
NEW CLEANER SIZE: 1
ELAPSED TIME: 50.75699996948242
==========================
@@ -91,0 +91,3 @@
+ {
+     FilterContainer.getInstance().clear();
+ }
